src/Service.Tests/GraphQLBuilder/MultiSourceBuilderTests.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.IO.Abstractions.TestingHelpers;
+using System.Threading.Tasks;
+using Azure.DataApiBuilder.Auth;
+using Azure.DataApiBuilder.Config;
+using Azure.DataApiBuilder.Core.Configurations;
+using Azure.DataApiBuilder.Core.Resolvers.Factories;
+using Azure.DataApiBuilder.Core.Services;
+using Azure.DataApiBuilder.Core.Services.MetadataProviders;
+using HotChocolate.Language;
+using Microsoft.Extensions.Logging;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Azure.DataApiBuilder.Service.Tests.GraphQLBuilder
+{
+    [TestClass]
+    public class GraphQLSchemaBuilderTests
+    {
+        [DataTestMethod]
+        public async Task CosmosSchemaBuilderTestAsync()
```

Comment:
comment would be helpful to explain how this test validates cosmos schema building. e.g. ensures that the expected number of fields are present in the GraphQLSchemaCreator's generated schema.

Similar to Ani's question on the .gql file you provided, does this test depend on all that file's content be present? because customers are only required to provide a .gql file with limited contents (GraphQL Object Type definitions only). Will this test still work if that's the case?

Reply:
this test is only using the limited schema.gql already being used for cosmos tests in other folders. the other .gql i added is specific to multi-source query execution